### PR TITLE
🚀 Installation and publishing magic

### PR DESCRIPTION
**Pull request by semanticer**
## Summary
I will have to add some internal docs with all the private information etc but otherwise, everything is ready and the library is now published on maven central. 

I mostly followed [this tutorial](https://medium.com/mobile-app-development-publication/upload-to-mavencentral-made-easy-for-android-library-30d2b83af0c7).

We can improve this even further with bitrise publishing in the future.  

## Relationships

Closes https://github.com/toggl/android/issues/2135

### CODE REVIEW

File: komposable-architecture/build.gradle [hunk 1]
```diff
@@ -1,6 +1,36 @@
...
+    PUBLISH_VERSION = '0.1.0'
+    PUBLISH_ARTIFACT_ID = 'komposable-architecture'
+    PUBLISH_DESCRIPTION = 'Kotlin implementation of Point-Free\'s composable architecture'
+    PUBLISH_URL = 'https://github.com/toggl/komposable-architecture'
+    PUBLISH_LICENSE_NAME = 'Apache License'
+    PUBLISH_LICENSE_URL =
+            'https://github.com/toggl/komposable-architecture/blob/main/LICENSE'
+    PUBLISH_DEVELOPER_ID = 'toggl'
+    PUBLISH_DEVELOPER_NAME = 'Toggl Track'
+    PUBLISH_DEVELOPER_EMAIL = 'tomas.valenta@toggl.com'
```
Comment (semanticer) [inline komposable-architecture/build.gradle:18]: What email address should I use here @pdrj ?

**Comment by daividssilverio** [comment]
> I mostly followed this tutorial.

Which tutorial?

File: komposable-architecture/build.gradle (hunk 1 above)
Comment (daividssilverio) [reply komposable-architecture/build.gradle:18]: Hey, the developer entry accepts multiple developers o/ Let's add the whole team o/

**Review by daividssilverio — APPROVED** [review]
I mean, looks alright. Looks like we have what's needed for publishing the lib.
Not really sure on how to evaluate this.

**Comment by semanticer** [comment]
> Which tutorial?

Sorry, forgot to include the link
https://medium.com/mobile-app-development-publication/upload-to-mavencentral-made-easy-for-android-library-30d2b83af0c7

File: komposable-architecture/build.gradle (hunk 1 above)
Comment (pdrj) [reply komposable-architecture/build.gradle:18]: Regarding email, our goto is I think support@toggl.com 🤔 There used to be something for mobile team, but I don't know how to access it 😄 If we clear this with support (they would just need to let us know about incoming emails), I think the support one would work.

**Review by pdrj — APPROVED** [review]
Looks good, I haven't tested this though. (In case you want us to run some tests, let us know.)

I asked about the email here https://toggl.slack.com/archives/C8H3UCQ9W/p1633098594182200. Once support confirms it is fine, we can merge this.

File: komposable-architecture/build.gradle (hunk 1 above)
Comment (semanticer) [reply komposable-architecture/build.gradle]: > Hey, the developer entry accepts multiple developers o/ Let's add the whole team o/

I changed that to our team, for now, I'm not sure who to include but we can definitely do that later.

**Pull request merged**